go.mod

```diff
@@ -177,10 +177,10 @@ replace (
 	github.com/go-log/log => github.com/go-log/log v0.1.0
 	github.com/gorilla/websocket => github.com/gorilla/websocket v1.4.2
 	github.com/mtrmac/gpgme => github.com/mtrmac/gpgme v0.1.1
-	github.com/openshift/api => github.com/openshift/api v0.0.0-20230201213816-61d971884921 // release-4.13
+	github.com/openshift/api => github.com/openshift/api v0.0.0-20230223193310-d964c7a58d75 //release 4.14
 	github.com/openshift/client-go => github.com/openshift/client-go v0.0.0-20210521082421-73d9475a9142 // release-4.8
 	github.com/openshift/library-go => github.com/openshift/library-go v0.0.0-20230130232623-47904dd9ff5a // release-4.13
-	github.com/openshift/machine-config-operator => github.com/openshift/machine-config-operator v0.0.1-0.20210701174259-29813c845a4a // release-4.8
+	github.com/openshift/machine-config-operator => github.com/openshift/machine-config-operator v0.0.1-0.20230410170945-be515e40d1c8 // release-4.13
```

Comment:
should'nt this be aligned to 4.14 as well if we want to be consistent ?

Reply:
As I was updating NTO to release-4.14 I used the same MCO version that NTO 4.14 is using.

Reply:
can you please update that to 4.14?

Reply:
yeah sure, done.